tests/e2e/support/objects/app-files/resource/utils.ts

```diff
@@ -1,8 +1,9 @@
-import { errors, Page } from '@playwright/test'
+import { errors, Page, Locator } from '@playwright/test'
 import util from 'util'
 
-export const resourceNameSelector =
-  ':is(#files-space-table, .oc-tiles-item, #files-shared-with-me-accepted-section, .files-table) [data-test-resource-name="%s"]'
+const resourceNameSelector = '#files-space-table [data-test-resource-name="%s"]'
```

Comment:
Need to export this selector so that it can be imported from others. CI complains about it
```suggestion
export const resourceNameSelector = '#files-space-table [data-test-resource-name="%s"]'
```